src/components/CreateNewProjectWizard/MainPane.js

```diff
@@ -75,39 +76,12 @@ class MainPane extends PureComponent<Props> {
 
               {currentStepIndex > 0 && (
                 <FadeIn>
-                  <FormField
-                    label="Project Type"
-                    isFocused={activeField === 'projectType'}
-                  >
-                    <ProjectTypeTogglesWrapper>
-                      {/* Todo: Make it easier to add new flows - e.g. map over an array to generate the UI*/}
-                      <ButtonWithIcon
-                        showStroke={projectType === 'create-react-app'}
-                        icon={<ReactIcon src={reactIconSrc} />}
-                        onClick={() =>
-                          this.updateProjectType('create-react-app')
-                        }
-                      >
-                        Vanilla React
-                      </ButtonWithIcon>
-                      <Spacer inline size={10} />
-                      <ButtonWithIcon
-                        showStroke={projectType === 'gatsby'}
-                        icon={<GatsbyIcon src={gatsbyIconSrc} />}
-                        onClick={() => this.updateProjectType('gatsby')}
-                      >
-                        Gatsby
-                      </ButtonWithIcon>
-                      <Spacer inline size={10} />
-                      <ButtonWithIcon
-                        showStroke={projectType === 'nextjs'}
-                        icon={<NextjsIcon src={nextjsIconSrc} />}
-                        onClick={() => this.updateProjectType('nextjs')}
-                      >
-                        Next.js
-                      </ButtonWithIcon>
-                    </ProjectTypeTogglesWrapper>
-                  </FormField>
+                  <ProjectTypeSelection
+                    projectType={projectType}
+                    onSelect={selectedProjectType =>
```

Comment:
Can we change `onSelect` to `onProjectTypeSelect` so that it’s consistent with `ProjectIconSelection`?

Reply:
Yes, that's better. I'll rename it.